cmd/tailscale/cli/status.go

```diff
@@ -221,9 +222,43 @@ func runStatus(ctx context.Context, args []string) error {
 		outln()
 		printHealth()
 	}
+	printFunnelStatus(ctx)
 	return nil
 }
 
+// printFunnelStatus prints the status of the funnel, if it's running.
+// Prints nothing if the funnel is not running.
+func printFunnelStatus(ctx context.Context) {
+	sc, err := localClient.GetServeConfig(ctx)
+	if err != nil {
+		outln()
+		printf("# Funnel:\n")
+		printf("#     - Unable to Funnel status: %v\n", err)
+	}
+	if sc == nil || len(sc.AllowFunnel) == 0 {
+		return
+	}
+	outln()
+	printf("# Funnel on:\n")
+	for hp, on := range sc.AllowFunnel {
+		if !on {
+			continue
+		}
```

Comment:
add bool method to ServeConfig that reports whether funnel is on and loop over map values looking for a true

otherwise line 242 can say "Funnel on:" if there's only a false map entry

Reply:
You're right. Shouldn't happen since we `delete()`. Done.